SPECS/keepalived/keepalived.spec

```
@@ -74,22 +92,26 @@ fi
 %systemd_postun_with_restart keepalived.service
 
 %files
-%defattr(-,root,root)
-%license COPYING
-%doc %{_docdir}/%{name}/README
-%{_sbindir}/%{name}
-%{_bindir}/genhash
-%{_unitdir}/%{name}.service
-%config(noreplace) %{_sysconfdir}/sysconfig/%{name}
-%config(noreplace) %{_sysconfdir}/%{name}/%{name}.conf
+%defattr(-, root, root, -)
+%doc AUTHOR ChangeLog CONTRIBUTORS COPYING README TODO
```

Please add licensing-related files with the `%license` macro.

```suggestion
%license COPYING
%doc AUTHOR ChangeLog CONTRIBUTORS README TODO
```